File: Cocoa Programming/Cocoa Pr L62 ObjC/Cocoa Pr L62 ObjC/SecondViewController.m
//
//  SecondViewController.m
//  Cocoa Pr L62 ObjC
//
//  Created by Michele Galvagno on 09/06/22.
//

#import "SecondViewController.h"

@interface SecondViewController ()
@property (weak) IBOutlet NSTextField *nameLabel;

@end

@implementation SecondViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    _nameLabel.stringValue = self.representedObject;
}

@end

File: Cocoa Programming/Cocoa Pr L62 Storyboard/Cocoa Pr L62 Storyboard/SecondViewController.swift
//
//  SecondViewController.swift
//  Cocoa Pr L62 Storyboard
//
//  Created by Michele Galvagno on 07/06/22.
//

import Cocoa

class SecondViewController: NSViewController {
    @IBOutlet weak var nameLabel: NSTextField!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        nameLabel.stringValue = representedObject as! String
    }
    
}
